atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -808,16 +805,14 @@ public Map<Cell, Value> get(TableReference tableRef, Map<Cell, Long> timestampBy
             ImmutableMap.Builder<Cell, Value> builder = ImmutableMap.builder();
             for (long ts : cellsByTs.keySet()) {
                 StartTsResultsCollector collector = new StartTsResultsCollector(metricsManager, ts);
-                try (CloseableTracer tracer = CloseableTracer.startSpan("loadWithTs")) {
-                    cellLoader.loadWithTs(
-                            "get",
-                            tableRef,
-                            cellsByTs.get(ts),
-                            ts,
-                            false,
-                            collector,
-                            readConsistencyProvider.getConsistency(tableRef));
-                }
+                cellLoader.loadWithTs(
```

Comment:
Why do we think that no longer tracing `loadWithTs` is valuable?

Reply:
For context -- this was added recently (https://github.com/palantir/atlasdb/pull/5466)

Reply:
Answer: it is, but `loadWithTs` is already wrapped in`CellLoader`, so these calls are useless!

Reply:
nit: could we rename the tracers in `CellLoader#loadWithTs` i.e. prefix them with `loadWithTs`

Reply:
I think you are asking for the spans to be named from "partitionByHost" to "loadWithTs.partitionByHost" for example.  If so, done!